jcl/src/jdk.management/share/classes/module-info.java.extra

```diff
@@ -21,7 +21,9 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 module jdk.management {
+	requires openj9.jvm;
```

Comment:
What motivates this requirement?

Reply:
This mxbean reuses Dump API, hence this module requires openj9.jvm

Reply:
We will need another strategy that doesn't necessitate this extra 'requires' clause. See https://github.com/ibmruntimes/openj9-openjdk-jdk9/pull/125.